stdlib/public/stubs/Stubs.cpp

```diff
@@ -172,13 +172,13 @@ static uint64_t swift_floatingPointToString(char *Buffer, size_t BufferLength,
 extern "C" uint64_t swift_float32ToString(char *Buffer, size_t BufferLength,
                                           float Value) {
   return swift_floatingPointToString<float>(Buffer, BufferLength, Value,
-                                            "%0.*g");
+                                            "%0.9g");
```

Comment:
By changing the format string, you are invoking undefined behavior when swift_snprintf_l is called within swift_floatingPointToString; the call now passes the wrong number of arguments.  The goal here is broadly fairly sensible, but instead of changing the format string, the more correct fix would be to replace numeric_limits<T>::digits10 with numeric_limits<T>::max_digits10.  It's worth noting that that's a C++11-ism, but I believe that's fine (someone else please confirm).